Source/App/EbAppConfig.h

```diff
@@ -10,309 +10,285 @@
-    uint32_t        encoder_bit_depth;
-    uint32_t        source_width;
-    uint32_t        source_height;
-
-    int64_t         frames_to_be_encoded;
-    int32_t         buffered_input;
-    unsigned char **sequence_buffer;
-
-    /*****************************************
-     * Coding Structure
-     *****************************************/
-    uint32_t        base_layer_switch_mode;
-    uint8_t         enc_mode;
-    int32_t         intra_period;
-    uint32_t        intra_refresh_type;
-    uint32_t        pred_structure;
-
-
-    /****************************************
-     * Quantization
-     ****************************************/
-    uint32_t        qp;
-
-    /****************************************
-     * Loop Filter
-     ****************************************/
-    uint32_t       loop_filter;
-
-    /****************************************
-     * ME Tools
-     ****************************************/
-    uint8_t        use_default_me_hme;
-    uint8_t        enable_hme_flag;
-
-    /****************************************
-     * ME Parameters
-     ****************************************/
-    uint32_t       search_area_width;
-    uint32_t       search_area_height;
-
-
-    /****************************************
-     * Rate Control
-     ****************************************/
-    uint32_t       rate_control_mode;
-    uint32_t       target_bit_rate;
-    uint32_t       max_qp_allowed;
-    uint32_t       min_qp_allowed;
-    /****************************************
-    * TUNE
-    ****************************************/
-    uint8_t        tune;
-
-    /****************************************
-     * Annex A Parameters
-     ****************************************/
-    uint32_t       profile;
-    uint32_t       level;
-
-    /****************************************
-     * On-the-fly Testing
-     ****************************************/
-    uint8_t        eos_flag;
-
-    /****************************************
-    * Optimization Type
-    ****************************************/
-    uint8_t        asm_type;
-
-    /****************************************
-     * Computational Performance Data
-     ****************************************/
-    EbPerformanceContext  performance_context;
-
-    /****************************************
-    * Instance Info
-    ****************************************/
-    uint32_t        channel_id;
-    uint32_t        active_channel_count;
-    int32_t         target_socket;
-    uint32_t        logical_processors;
-    uint8_t         stop_encoder;         // to signal CTRL+C Event, need to stop encoding.
-
-    uint64_t        processed_frame_count;
-    uint64_t        processed_byte_count;
-
+typedef struct EbPerformanceContext {
+
+  /****************************************
+   * Computational Performance Data
+   ****************************************/
+  uint64_t lib_start_time[2];    // [sec, micro_sec] including init time
```

Comment:
Majority code use 4 space indent. Could you also use 4 space? Thx.

Reply:
oops, clang format used default, going to revert